routes/api/contacts.js

```diff
@@ -1,25 +1,95 @@
-const express = require('express')
+const express = require("express");
+const router = express.Router();
 
-const router = express.Router()
+const { v4 } = require("uuid");
+const Joi = require("joi");
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+const { listContacts, getContactById, removeContact, addContact, updateContact } = require("../../models/contacts.js");
```

Comment:
Importy powinny mieć max 80 char length i jeśli są wieksze to ułozone alfabetycznie jeden pod drugim.